README.md

```diff
@@ -1,17 +1,33 @@
 # Package Support Framework
-This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as packaged apps.
+This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as MSIX packaged apps.
+
+Here are some common examples where you can find the PSF useful:
+	1. Your app can't find some .dlls when launched, you may need to set your current working directory.
+	You can learn about the required current working directory in the original shortcut before you converted to MSIX.
+	2. The app writes into the install folder. You will typically see it by "Access Denied" errors in 

```

Comment:
Can we have a link to what an ACCESS DENIED error looks like in procmon?

Reply:
Sure will do